packages/dashboard/src/components/DashboardGridLayout/AddCardPanel/AddCardPanel.types.ts

```diff
@@ -0,0 +1,48 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { IDGLCard } from '../../../index';
+
+export interface IAddCardPanelProps {
+  /**
+   * The title for the add card panel
+   */
+  header?: string;
+
+  /**
+   * Prop to open the add card panel
+   */
+  isOpen?: boolean;
+
+  /**
+   * The list of cards to render in the add card panel
+   */
+  cards?: IDGLCard[];
+
+  /**
+   * The callback method to switch between add card panel and the dashboard grid layout
+   */
+  switchCard?: (cardId: string) => void;
+}
```

Comment:
what is this used for?

Reply:
This is used for internal purpose. This callback is called when a user clicks on a card in add card panel and the card needs to be switched by DGLWithAddCardPanel code to dashboard.